### ISSUE TITLE
logo proposal

### ISSUE THREAD

**Issue by dee-y** [stimonm/Bike3S#35]
Hi, i am a graphic designer. Would you be interested i contribute a free logo to your project?

**Comment by cruizba** [comment]
Of course! You're free to suggest and create a logo for the simulator 👍
Thanks for the interest 🐼
